influxql/INFLUXQL.md

```diff
@@ -229,6 +229,7 @@ ALTER RETENTION POLICY policy1 ON somedb DURATION 1h REPLICATION 4
 
 ```
 create_continuous_query_stmt = "CREATE CONTINUOUS QUERY" query_name on_clause
+                               [ "RESAMPLE" [ "EVERY" duration_lit ] [ "FOR" duration_lit ] ]
```

Comment:
I attempted to make this more clear, but it's very difficult to do a "at least one, but in this order" in BNF since that's not really a thing in context free grammars. It's possible in a context-free grammar, but it looks weird since you basically have to say `S = EF | E | F`.